mistune/renderers.py

```diff
@@ -245,19 +245,10 @@ def codespan(self, text):
         return f'`{text}`'
 
     def strong(self, text):
-        return '**' + text + '**'
-
-    def table(self, table):
-        from bs4 import BeautifulSoup
-        from tabulate import tabulate
-        soup = BeautifulSoup('<table>' + table + '</table>', 'html.parser')
-        table = soup.find('table')
-        thead = table.find('thead')
-        headers = [cell.text for cell in thead.find_all('th')]
-        tbody = table.find('tbody')
-        rows = [[cell.text for cell in row.find_all('td')]
-                 for row in tbody.find_all('tr')]
-        return tabulate(rows, headers=headers) + self.linebreak() + self.linebreak()
+        return '\033[1m' + text + '\033[0m'
```

Comment:
That change seems unrelated?

Reply:
This change is necessary to yield proper bold setting for terminal output. Would you prefer a separate pull request for this change?